stable/yugaware/templates/configs.yaml

```diff
@@ -32,6 +32,8 @@ data:
     pidfile.path = "/dev/null"
     play.evolutions.enabled=false
     play.modules.enabled += "org.flywaydb.play.PlayModule"
+    # play.logger.includeConfigProperties=true
+    # log.override.path = "/opt/yugaware_data/logs"
```

Comment:
do you want to still keep these commented? setting java properties seems to be more involved, right?

Reply:
If we uncomment these, then we will see blank `/logs` page. Basically we need, https://github.com/yugabyte/yugabyte-db/pull/6687

Reply:
Makes sense. Now that the other change is in, let's uncomment and merge this PR too.